fetch-in-react/my-app/src/Users.jsx

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable @typescript-eslint/no-unused-vars -- Remove me */
+import { useEffect, useState } from 'react';
+import UserDetails from './UserDetails';
+import UserCard from './UserCard';
+import './Users.css';
+
+export default function Users() {
+  const [isLoading, setIsLoading] = useState(true);
+  const [error, setError] = useState();
+  const [users, setUsers] = useState([]);
+  const [user, setUser] = useState();
+
+  /* your code here (hint: useEffect) */
+  useEffect(() => {
+  const response = new Request('https://jsonplaceholder.typicode.com/users');
+  fetch(response)
+    .then((response) => {return response.json()})
```

Comment:
FYI: we can simplify this code to make advantage of arrow functions implict returns 

`.then((response) =>  response.json())`